### PR TITLE
[PUBDEV-7283] Save to hive - parquet support, external tables

### PR DESCRIPTION
**Pull request by honzasterba**

### CODE REVIEW

**Comment by honzasterba** [comment]
@hannah-tillman please review the docs

File: h2o-docs/src/product/data-munging/downloading-data.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+Downloading data
+----------------
+
+Sometimes it is desirable to download data from the H2O cluster. For example when computing model predictions, it might be desirable to save these predictions for later.
+
+Download to local memory
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+H2O has functions like ``as_data_frame`` and ``get_frame_data`` in Python and ``as.data.frame`` in R that allow to download the data directly into the client program memory.
```
Comment (hannah-tillman) [inline h2o-docs/src/product/data-munging/downloading-data.rst:9]: "...that allow to download the data..." -> "...that allow **you** to download the data..."

File: h2o-docs/src/product/data-munging/downloading-data.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+**Note**: For very large data this might not be feasible since the whole frame is downloaded as CSV into the client program memory.
+
+.. tabs::
+   .. code-tab:: r R
+	
+        library(h2o)
+		h2o.init()
```
Comment (hannah-tillman) [inline h2o-docs/src/product/data-munging/downloading-data.rst:17]: Please fix the indentation here and in the python example

File: h2o-docs/src/product/data-munging/downloading-data.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Save to a file system
+~~~~~~~~~~~~~~~~~~~~~
+
+The export file function can be used to save the data to an arbitrary location. The location has to be one that the server has access to, so either the server filesystem or a distributed filesystem like HDFS or S3. This function will save the data in CSV format.
+
+.. tabs::
+   .. code-tab:: r R
+	
+        library(h2o)
+		h2o.init()
```
Comment (hannah-tillman) [inline h2o-docs/src/product/data-munging/downloading-data.rst:38]: Please fix the indentation here and in the python example

File: h2o-docs/src/product/data-munging/downloading-data.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Save as a Hive table
+~~~~~~~~~~~~~~~~~~~~
+
+When running on Hadoop H2O can also export data into Hive tables. In order to do so the user running the H2O cluster must have the privileges to create new Hive tables. The user can specify the table name and storage format (currently supported are ``csv`` and ``parquet``) as well as table location for external tables.
+
+.. tabs::
+   .. code-tab:: r R
+	
+        library(h2o)
+		h2o.init()
```
Comment (hannah-tillman) [inline h2o-docs/src/product/data-munging/downloading-data.rst:58]: Please fix the indentation here and for the python example

File: h2o-docs/src/product/data-munging/downloading-data.rst [hunk 5]
```diff
@@ -0,0 +1,72 @@
+Downloading data
+----------------
+
+Sometimes it is desirable to download data from the H2O cluster. For example when computing model predictions, it might be desirable to save these predictions for later.
```
Comment (hannah-tillman) [inline h2o-docs/src/product/data-munging/downloading-data.rst:4]: minor (add comma): "For example,"

File: h2o-docs/src/product/data-munging/downloading-data.rst [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+   
+		import h2o
+		h2o.init()
+        iris_hex = h2o.import_file("iris/iris_wheader.csv")
+        h2o.export_file(iris_hex, path = "/tmp/pred.csv", force = True)
+
+Save as a Hive table
+~~~~~~~~~~~~~~~~~~~~
+
+When running on Hadoop H2O can also export data into Hive tables. In order to do so the user running the H2O cluster must have the privileges to create new Hive tables. The user can specify the table name and storage format (currently supported are ``csv`` and ``parquet``) as well as table location for external tables.
```
Comment (hannah-tillman) [inline h2o-docs/src/product/data-munging/downloading-data.rst:52]: minor (add comma x2): 
1) When running on Hadoop,"
2) "In order to do so,"

**Pull request merged**